libraries/standard/mqtt/cbmc/proofs/MQTT_SerializeConnect/MQTT_SerializeConnect_harness.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file MQTT_SerializeConnect_harness.c
+ * @brief Implements the proof harness for MQTT_SerializeConnect function.
+ */
+#include "mqtt.h"
+#include "mqtt_cbmc_state.h"
+
```

Comment:
Are these defines below proof assumptions?  Would it make sense to define these in the Makefile?  Or at least #ifdef/#endif these defines so they can be overridden from the Makefile.  The most important thing, though, it to have one system for collecting proof assumptions in one place and use it everywhere.

Reply:
Yes thanks for the advice. I left these during experimentation and I am not using them, so I will delete them.